timed/employment/filters.py

```diff
@@ -52,14 +52,20 @@ class Meta:
 class UserFilterSet(FilterSet):
     active      = NumberFilter(field_name='is_active')
     supervisor  = NumberFilter(field_name='supervisors')
-    is_reviewer = NumberFilter(method='filter_is_reviewers')
-    is_supervisor = NumberFilter(method='filter_is_supervisors')
-
-    def filter_is_reviewers(self, queryset, name, value):
-        return queryset.filter(pk__in=User.objects.all_reviewers())
-
-    def filter_is_supervisors(self, queryset, name, value):
-        return queryset.filter(pk__in=User.objects.all_supervisors())
+    is_reviewer = NumberFilter(method='filter_is_reviewer')
+    is_supervisor = NumberFilter(method='filter_is_supervisor')
+
+    def filter_is_reviewer(self, queryset, name, value):
+        if value:
+            return queryset.filter(pk__in=User.objects.all_reviewers())
+        else:
```

Comment:
This looks functional good now. Just one small thing: I usually try to avoid `else` whenever possible not to complicate the code.

This else could actually be easily removed as the return statement above already exist the method. Same below.